components/imageproc/src/format.rs

```diff
@@ -2,6 +2,8 @@ use errors::{anyhow, Result};
 use std::hash::{Hash, Hasher};
 
 const DEFAULT_Q_JPG: u8 = 75;
+const DEFAULT_Q_AVIF: u8 = 70;
+const DEFAULT_S_AVIF: u8 = 10;
```

Comment:
We should probably stop using single letter, I only understood S was for speed by looking a comment below

Reply:
I agree. I matched what was there, but I will change them to "QUALITY" and "SPEED".

Reply:
`ravif` (the crate behind `image`'s AVIF implementation) uses different defaults (quality 80, speed 5); any interest in switching the defaults in Zola?

https://github.com/kornelski/cavif-rs/blob/ed676dd1a3d9726b740ad679843bad55d3a84ebd/ravif/src/av1encoder.rs#L82-L84

"Fast but crappy" doesn't strike me as a great default speed, and (anecdotally) quality 70 is disappointing. Feel free to play around with the options on this branch, though, and see what you think.

Reply:
We can switch to their default